horace_core/sqw/file_io/@sqw_file_interface/sqw_file_interface.m

```diff
@@ -109,6 +109,7 @@
                 fseek(obj.file_id,0,'eof');
                 pos  = ftell(obj.file_id);
             end
+            pos = uint64(pos);
```

Comment:
because unit64 addresses 2^64 pixels and double -- 2^52 May be a bit overreaction but better think about it well in advance. At least in file format where it is difficult to change.